manifest.php

```diff
@@ -44,7 +44,7 @@
     'label' => 'Proctoring',
     'description' => 'Proctoring for deliveries',
     'license' => 'GPL-2.0',
-    'version' => '6.0.0',
+    'version' => '6.1.0',
```

Comment:
As there is an API dependency to taoQtiTest you should also update the required version, below. Otherwise the runtime could fail as the API related to the extended time could be missing.